### PR TITLE
Fix for CLOUDSTACK-9253

### PR DESCRIPTION
**Pull request by pfarmer**
Fix for CLOUDSTACK-9253, the instructions listed [here](https://hub.docker.com/r/cloudstack/simulator/) should now work correctly.

### CODE REVIEW

**Comment by rhtyd** [comment]
@pfarmer thanks, please rebase against latest master, squash commits to a single commit; the version need to be changes to 4.9.0

tag:easypr

File: tools/docker/Dockerfile [hunk 1]
```diff
@@ -20,7 +20,7 @@
 FROM ubuntu:14.04
 
 MAINTAINER "Apache CloudStack" <dev@cloudstack.apache.org>
-LABEL Vendor="Apache.org" License="ApacheV2" Version="4.6.0"
+LABEL Vendor="Apache.org" License="ApacheV2" Version="4.8.0"
```
Comment (rhtyd) [inline tools/docker/Dockerfile]: Version needs to be changed to 4.9.0

**Comment by bvbharatk** [comment]
### ACS CI BVT Run
 **Sumarry:**
 Build Number 147
 Hypervisor xenserver
 NetworkType Advanced
 Passed=68
 Failed=5
 Skipped=3

_Link to logs Folder (search by build_no):_ https://www.dropbox.com/sh/yj3wnzbceo9uef2/AAB6u-Iap-xztdm6jHX9SjPja?dl=0


**Failed tests:**
* test_vpc_vpn.py

 * ContextSuite context=TestRVPCSite2SiteVpn>:setup Failing since 12 runs

 * ContextSuite context=TestVpcRemoteAccessVpn>:setup Failing since 12 runs

 * ContextSuite context=TestVpcSite2SiteVpn>:setup Failing since 12 runs

* test_volumes.py

 * test_06_download_detached_volume Failing since 2 runs

* test_vm_life_cycle.py

 * test_10_attachAndDetach_iso Failing since 2 runs


**Skipped tests:**
test_vm_nic_adapter_vmxnet3
test_static_role_account_acls
test_deploy_vgpu_enabled_vm

**Passed test suits:**
test_deploy_vm_with_userdata.py
test_affinity_groups_projects.py
test_portable_publicip.py
test_over_provisioning.py
test_global_settings.py
test_scale_vm.py
test_service_offerings.py
test_routers_iptables_default_policy.py
test_routers.py
test_reset_vm_on_reboot.py
test_snapshots.py
test_deploy_vms_with_varied_deploymentplanners.py
test_login.py
test_list_ids_parameter.py
test_public_ip_range.py
test_multipleips_per_nic.py
test_regions.py
test_affinity_groups.py
test_network_acl.py
test_pvlan.py
test_nic.py
test_deploy_vm_root_resize.py
test_resource_detail.py
test_secondary_storage.py
test_disk_offerings.py

**Comment by rhtyd** [comment]
Thanks @pfarmer for the pull request, given it's not applicable to master/4.11 anymore I'll close this. Kindly re-submit a new PR. Thanks again for your contribution.

**Pull request closed without merging**